src/test/java/user_interact_view_showcase/FriendManagerFileViewShowcase.java

```diff
@@ -0,0 +1,109 @@
+package user_interact_view_showcase;
+
+import abr.user_interact_abr.manage_friend_request_abr.FriendManagerDsGateway;
+import abr.user_interact_abr.manage_friend_request_abr.FriendManagerInputBoundary;
+import abr.user_interact_abr.manage_friend_request_abr.FriendManagerOutputBoundary;
+import abr.user_interact_abr.manage_friend_request_abr.FriendManagerPresenter;
+import abr.user_interact_abr.manage_friend_request_abr.deleting_attempt_abr.DeleteFriendOrDenyFriendRequest;
+import abr.user_interact_abr.manage_friend_request_abr.sending_or_accepting_attempt_abr.SendFriendRequest;
+import abr.user_interact_abr.show_friend_list_abr.OrderFriendList;
+import abr.user_interact_abr.show_friend_list_abr.OrderFriendListInputBoundary;
+import ds.user_database.UserFileReader;
+import ds.user_database.UserFileWriter;
+import ds.user_interact_ds.FriendManagerFileDsGateway;
+import entities.user_entities.CommonUser;
+import entities.user_entities.User;
+import framework.user_interact_screen.friend_manager_screen.FriendListView;
+import interface_adaptors.user_interact_ia.DeleteFriendOrDenyFriendRequestController;
+import interface_adaptors.user_interact_ia.SendFriendRequestController;
+import interface_adaptors.user_interact_ia.ShowFriendListController;
+import interface_adaptors.user_interact_ia.TempFriendListObservable;
```

Comment:
Make sure your imports are clean and concise. Also, I see that you are importing entities... Is this intended?

Reply:
this is not in the layers, technically some sort of main for testing my own UI. I'll delete this once we finished the main page UI.

Reply:
Ok I understand, I see no issue then 👍